### PR TITLE
Fix issue #1169 - More localization

### LINKED ISSUE #1169: Some strings i can't translate on Transifex
It seems that some string are not on Transifex for me to translate.

Like the three buttons on the bottom "Discover", "Make", "My Apps".
The menu "Back to My Apps", "Share", "Delete".
"App Name", "From Scratch", "No data has been stored yet!", etc.

### PR DESCRIPTION
**Pull request by alicoding**
closes #1169

### CODE REVIEW

**Comment by alicoding** [comment]
@thisandagain R?

File: locale/en_US/mobile-appmaker.json [hunk 1]
```diff
@@ -111,6 +119,18 @@
...
+    "message": "No data has been stored yet!",
+    "description": "No data has been stored yet!"
+  },
+  "noDataInfoSub": {
+    "message": "If you'd like to let users share data with you, try adding a Counter and a Submit button to your app.",
+    "description": "If you'd like to let users share data with you, try adding a Counter and a Submit button to your app."
+  },
+  "Publishing...": {
+    "message": "Publishing...",
+    "description": "Publishing... status"
```
Comment (k88hudson) [inline locale/en_US/mobile-appmaker.json]: Can you update this to say "Shows up below header when the app is being published/ a publish is in progress

**Comment by alicoding** [comment]
@k88hudson R? [[flagged!]](http://plzkthx.herokuapp.com)

**Comment by alicoding** [comment]
Not sure why travis is failing on me here. Ran gulp test and seeing this 

```
  1) export.js should copy publish assets:
     AssertionError: false == true
      at /Users/ali/projects/webmaker-suite/webmaker-app/test/unit/export.js:37:17
      at DestroyableTransform.<anonymous> (/Users/ali/projects/webmaker-suite/webmaker-app/export.js:9:9)
      at DestroyableTransform.emit (events.js:117:20)
      at /Users/ali/projects/webmaker-suite/webmaker-app/node_modules/gulp/node_modules/vinyl-fs/node_modules/through2/node_modules/readable-stream/lib/_stream_readable.js:965:16
      at process._tickDomainCallback (node.js:486:13)

```

**Pull request merged**